docs/ibc/spec.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: "Core Protocol Spec"
+description: "The foundation of the Inter-Blockchain Communication Protocol"
+---
+<footnote>
+For legacy IBC implementations, refer to the [IBC‑go docs](https://ibc.cosmos.network/main/).
+</footnote>
+
+import { footnote } from '/snippets/footnote.mdx'
+
+
+The latest version of Inter‑Blockchain Communication Protocol (**IBC**) enables token transfers across multiple blockchain environments through a standard set of components, interfaces, and a defined packet flow.
+
+
+### Core Data Structures
+
+The **`Packet`** is the primary container for cross-chain communication. Each packet wraps one or more application-specific **`Payload`** objects.
+
+<CodeGroup>
+
+```typescript packet.ts
+// The main data container sent between chains
+interface Packet {
+  sourceClientId: bytes; // Client ID for destination chain (stored on source)
+  destClientId: bytes;   // Client ID for source chain (stored on destination)
+  sequence: uint64;      // Monotonically increasing nonce for ordering
+  timeoutTimestamp: uint64; // UNIX timestamp (seconds) when the packet expires
+  data: Payload[];       // List of application payloads
+}
+
+// Application-specific data and routing information
+interface Payload {
+sourcePort: bytes; // Identifies sending application module
+destPort: bytes;   // Identifies receiving application module
+version: string;   // App-specific version for interpretation
+encoding: string;  // MIME-type for decoding value
+value: bytes;    // Opaque application data
+}
+
+```
+
+```json packet.json
+{
+  "sourceClientId": "07-tendermint-0",
+  "destClientId": "07-tendermint-0",
+  "sequence": 1,
+  "timeoutTimestamp": 1617348393,
+  "data": [
+    {
+      "sourcePort": "transfer",
+      "destPort": "transfer",
+      "version": "ics20-1",
+      "encoding": "application/json",
+      "value": "<base64-encoded application data>"
+    }
+  ]
+}
+```
+
+</CodeGroup>
+
+
+## Key Components
+
+<Accordion title="Client (ICS‑02) – Light Client">
+An on‑chain light client that verifies the state of a counter‑party chain using cryptographic proofs against a trusted consensus state.
```

Comment:
Nit: ICS02 specifies how light clients should behave, as well as how the protocol itself should handle client creation (adding new light clients) and client routing (making sure updateClient, verify* goes to the correct light client).